### ISSUE TITLE
Media: Unable to upload .ogg audio files to the Media Library

### ISSUE THREAD

**Issue by thehenrybyrd** [wordpress-mobile/WordPress-iOS#15904]
### Expected behavior
In testing the new Audio Block, I expected to be able to upload any audio file from my phone to the WordPress app, in particular .mp3, .m4a, .wav and .ogg.
I expected to be able to upload a .ogg directly to the Media Library.

### Actual behavior
I was unable to upload a .ogg file in any way, including directly to the Media Library.

### Steps to reproduce the behavior

1. Get a .ogg file and get it onto your iPhone. I found one from WikiMedia and Airdropped it to myself, saving it in iOS Files. 
2. From Media, tap “+” and then “Other Apps” and try to select the .ogg file from within iOS Files.

While it is possible to add a .ogg file to the Media Library from within a web browser, it appears greyed-out when trying to add it from WPiOS.

![GreyOgg](https://user-images.githubusercontent.com/15107387/108184014-62c00200-7113-11eb-92a2-2833217b28ee.jpeg)

**Comment by hypest** [comment]
👋 @thehenrybyrd , took the liberty of specifying in the ticket title that this is about the Media Library, mostly to clearly differentiate it from https://github.com/wordpress-mobile/gutenberg-mobile/issues/3165. Let me know if there's more nuance and the new title is not appropriate, thanks!

**Comment by sla8c** [comment]
This is resolved now.  Tested on v 18.9

I’ve uploaded an ogg file from WPiOS and I confirmed it existed via wordpress.com + desktop browser.

I also tested the full cycle end to end so I downloaded the same ogg file from desktop browser WordPress Media to my desktop.  The file was intact and played audio nicely i.e. the downloaded media file played nicely so the upload from WPiOS was successful

A couple of screenshots
1. .ogg file no longer greyed out in FilePicker screen
![image](https://user-images.githubusercontent.com/88816724/151002949-d53cde59-f01e-441b-9276-398b7546bf09.png)

A second screenshot confirming .ogg file extension
![image](https://user-images.githubusercontent.com/88816724/151003049-065e0605-0e47-44f0-88e2-51ac2c6b2d1e.png)


2. .ogg file exists on wordpress.com + desktop browser after upload
![image](https://user-images.githubusercontent.com/88816724/151000947-278728a4-7c93-4705-a381-20f559baef49.png)

3. downloaded this file from wordpress.com + desktopbrowser + playing audio on computer + works
